rmf_fleet_adapter/src/rmf_fleet_adapter/agv/test/MockAdapter.cpp

```diff
@@ -141,13 +141,13 @@ void MockAdapter::stop()
 //==============================================================================
 void MockAdapter::request_delivery(const rmf_task_msgs::msg::Delivery& request)
 {
-  rmf_fleet_adapter::agv::request_delivery(request, _pimpl->fleets);
+  // rmf_fleet_adapter::agv::request_delivery(request, _pimpl->fleets);
```

Comment:
We should at least put in some TODOs to fix up these functions whose implementations are now dead. There may be tests that depend on them. Or if there aren't tests that depend on them, then the functions should be removed entirely.

Reply:
Added TODOs https://github.com/osrf/rmf_core/pull/217/commits/3beae9802bb5d8baa5559ef0bcef98fcc3cda408